kDriveCore/Data/UploadQueue/Operation/UploadOperation.swift

```diff
@@ -477,39 +480,49 @@ public final class UploadOperation: AsynchronousOperation, UploadOperationable {
         }
     }
 
-    /// Propagate the newly uploaded DriveFile into the specialized Realm
+    /// Propagate the newly uploaded DriveFile / File into the specialized Realms
     func handleDriveFilePostUpload(_ driveFile: File) throws {
-        var driveId: Int?
-        var userId: Int?
-        var relativePath: String?
-        var parentDirectoryId: Int?
+        let readOnlyFile = try readOnlyFile()
+        let driveId = readOnlyFile.driveId
+        let userId = readOnlyFile.userId
+
+        let driveFileManagerContext: DriveFileManagerContext
+        @InjectService var appContext: AppContextServiceable
+        switch appContext.context {
+        case .fileProviderExtension:
+            driveFileManagerContext = .fileProvider
+        case .app, .shareExtension, .actionExtension:
+            // TODO: fix for `shared with me`
```

Comment:
Also `share with me` changes

Reply:
DriveFileManager with sharedWithMe context is now correctly returned. We should call `instanceWith` only to force fileManager context

Reply:
Ok changed in the last commits, will ask to to check it out.